app/proxy_pages.rb

```diff
@@ -10,9 +10,7 @@ def self.resources
 
   def self.api_docs
```

Comment:
`app_docs` already exists, so I've renamed the existing one to `app_overviews` (more appropriate, I reckon) and then applied the rename as suggested above.

Reply:
Thanks - I agree that's clearer. Looking again, I see there's a terminology conflict in the use of "docs": sometimes it refers to "docs in the repo"; sometimes to "manual pages". For me, the most confusing instance is `AppDocs.apps_with_docs` 🤯. One other potential smell here is the encapsulation of `AppDocs`, which seems to be leaking into `ProxyPages` - the latter is taking on some of the responsibility of the former by adapting the GitHub response. Could making more use of the `App` class would address both these issues?

(This was already an issue, so out-of-scope for this PR. I think it would be good to follow-up on, though, as I imagine it will be confusing for someone looking at this code in future.)

Reply:
`AppDocs.apps_with_docs` has never sat right with me! Agreed, we can refactor that out when we remove the `consume_docs_folder` functionality.

I seem to remember running into serialisation-related issues when delegating some of ProxyPage's concerns to the App, but will give this another go when that refactor PR comes. 👍